src/firmware_new/boards/frankie_v1/udp_multicast.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+/*
+ * Join the multicast group to receive packets from AI.
```

Comment:
Comment should maybe specify it starts up a new thread to handle things asyc?

Reply:
fixed